roles/chameleon_mariadb/defaults/main.yml

```diff
@@ -1,6 +1,9 @@
 ---
 backup_database_user: backup
 
+deployment_dir: hostvars[groups['mariadb'][0]]['deployment_dir']
```

Comment:
I'm not sure why these variables are needed, couldn't it refer to just the value of `deployment_dir` and `site_config_dir`? I don't think the various hosts are setting different values for these.